tests/integration/conftest.py

```diff
@@ -127,27 +127,27 @@ def _f(**kwargs: Any) -> OrchestratorConfig:
                 ResourcePoolType(
                     cpu=1.0,
                     available_cpu=1.0,
-                    memory_mb=2048,
-                    available_memory_mb=2048,
+                    memory=2048 * 10**6,
+                    available_memory=2048 * 10**6,
                 ),
                 ResourcePoolType(
                     cpu=1.0,
                     available_cpu=1.0,
-                    memory_mb=2048,
-                    available_memory_mb=2048,
+                    memory=2048 * 10**6,
+                    available_memory=2048 * 10**6,
                     is_preemptible=True,
                 ),
                 ResourcePoolType(
                     cpu=100,
                     available_cpu=100,
-                    memory_mb=500_000,
-                    available_memory_mb=500_000,
+                    memory=500 * 10**6_000,
```

Comment:
Is not `10**6_000` too large?

Reply:
This should be `500_000 * 10**6`. This is find-replace generated typo, thank you.